Sources/Apollo/NetworkTransport.swift

```diff
@@ -5,28 +5,30 @@ public protocol NetworkTransport: class {
   
   /// Send a GraphQL operation to a server and return a response.
   ///
+  /// Note: The `clientName` and `clientVersion` should be sent with any URL request which needs headers so your client can be identified by tools like Apollo Graph Manager. The `addClientHeaders` method is provided below to do this for you automatically, and this is handled for you by batteries-included versions of `NetworkTransport`. 
```

Comment:
```suggestion
  /// Note: The sender should properly set the headers `apollographql-client-name` and `apollographql-client-version` when sending the related request. This is so that Graph Manager can properly attribute traffic.`. 
```

Reply:
I'm going to decline this suggestion - the header names are not what specifically need to be set by the client, since those are included in the default implementation. It's the actual values which need to be set. I'll see if I can make this clearer though.